api/serializers/student.py

```diff
@@ -13,6 +13,10 @@
 
 
 class StudentWriteSerializer(serializers.ModelSerializer[Student]):
+    # For Student, we need Student.id (and not just personal_info.id) so we can pass it
+    # from the bot to the backend when we send the enrollment test.
+    internal_id = serializers.IntegerField(source="pk", read_only=True)
```

Comment:
Why not just "id"? Or even "personal_info_id" since we use it as an id. Just to be consistent and don't rename anything.